js/functional/fibonacci.js

```diff
@@ -6,8 +6,26 @@ import { parseNumber } from "../helper.js";
  * @param {number} sequence
  */
 function generateFibonacciUsingLoop(sequence) {
-  // write your code here
-  return [];
+  const fibonacciArray = [];
+
+  if (sequence <= 0) {
+    return fibonacciArray;
+  }
+  if (sequence >= 1) {
+    fibonacciArray.push(0);
+  }
+  if (sequence >= 2) {
+    fibonacciArray.push(1);
+  }
```

Comment:
You should give a line break (enter) between this code, so it will be easier for us to understand each part of the code.